### ISSUE TITLE
filter `ISoftDelete` has stopped working with the release of `3.3.0-rc.1`

### ISSUE THREAD

**Issue by 274188A** [abpframework/abp#5837]
This filter `ISoftDelete` has stopped working with the release of `3.3.0-rc.1`

Was working with previous versions

```
public async Task<List<Gender>> GetLookupListAsync()
{
	//Temporary disable the ISoftDelete filter
	using (_dataFilter.Disable<ISoftDelete>())
	{
		return await DbSet.OrderBy(nameof(Gender.Sort)).ToListAsync();		
	}
}

```

![image](https://user-images.githubusercontent.com/40014477/96259589-6a9a5900-0ff0-11eb-92f0-d9f4315a5461.png)
The IsDeleted should still return in the query - but does not.

**Comment by realLiangshiwei** [comment]
* Steps needed to **reproduce** the problem.
* Your DB Provider

**Comment by 274188A** [comment]
Apologies - more details below.  I put a break-point in the repository to see the results returned.  I expected 5 rows but only saw 4 (missing the one where `IsDeleted=true`).  So I do not think this is UI related.

### Stack
Blazor UI, EF, SQL Server, ABP 3.3.0-rc.1

### Steps
Using an FullAuditEntity<Guid> set one of the rows IsDeleted to true.  
Apply the filter in the repository.

**Expected Result** - see the entity where IsDeleted=true returned
**Actual Result** - entity where IsDeleted=true is not returned

 ### Log
>2020-10-16 20:38:03.051 +08:00 [INF] Executing endpoint 'Gp.AppServices.GenderAppService.GetLookUpListAsync (Gp.Application)'
2020-10-16 20:38:03.055 +08:00 [INF] Route matched with {action = "GetLookUpList", controller = "Gender", area = "", page = ""}. Executing controller action with signature System.Threading.Tasks.Task`1[Volo.Abp.Application.Dtos.PagedResultDto`1[Gp.Genders.GenderLookupDto]] GetLookUpListAsync(Volo.Abp.Application.Dtos.PagedAndSortedResultRequestDto) on controller Gp.AppServices.GenderAppService (Gp.Application).
2020-10-16 20:38:03.105 +08:00 [INF] Executing action method Gp.AppServices.GenderAppService.GetLookUpListAsync (Gp.Application) - Validation state: "Valid"
2020-10-16 20:38:03.108 +08:00 [INF] Entity Framework Core 3.1.8 initialized 'GpDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2020-10-16 20:38:03.277 +08:00 [INF] Executed DbCommand (1ms) [Parameters=[@__ef_filter__p_0='?' (DbType = Boolean)], CommandType='"Text"', CommandTimeout='30']
SELECT [g].[Id], [g].[CreationTime], [g].[CreatorId], [g].[DeleterId], [g].[DeletionTime], [g].[IsDeleted], [g].[LastModificationTime], [g].[LastModifierId], [g].[Name], [g].[Sort]
FROM [gp].[Genders] AS [g]
WHERE (@__ef_filter__p_0 = CAST(1 AS bit)) OR ([g].[IsDeleted] <> CAST(1 AS bit))
ORDER BY [g].[Sort]

**Comment by realLiangshiwei** [comment]
I will check it out

**Comment by realLiangshiwei** [comment]
Hi
I can't reproduce your problem. could you use the startup template to reproduce this problem and upload to GitHub repository. thanks.

![image](https://user-images.githubusercontent.com/16813853/96422511-b2a4c000-122a-11eb-963b-d36d161d3df9.png)

**Comment by jaszb** [comment]
I finally got t the bottom of the problem.  I created a project from the template and as you found there is no problem with Abp.

I had accidently used this interface:
![image](https://user-images.githubusercontent.com/32381192/96528546-9dc13e80-12b5-11eb-95ae-3ff17c4f5efe.png)

instead of this one:

![image](https://user-images.githubusercontent.com/32381192/96528574-ad408780-12b5-11eb-8b5b-9f17e5982f0c.png)


Happy to close this and thanks for your time once again!
Cheers,
john
